designer/server/src/models/forms/edit.js

```diff
@@ -0,0 +1,21 @@
+import * as create from '~/src/models/forms/create.js'
+
+/**
+ * @param {string} slug
+ * @param {Partial<FormMetadataInput>} [metadata]
+ * @param {ValidationFailure} [validation]
+ */
+export function organisationViewModel(slug, metadata, validation) {
+  return {
+    ...create.organisationViewModel(metadata, validation),
+    backLink: {
+      href: `/library/${slug}`
```

Comment:
We set the text usually, that alright?

```suggestion
      text: 'Back to form overview',
      href: `/library/${metadata.slug}`
```

Reply:
Done